### PR TITLE
[QEff. Finetune] : To plot correct loss values on tensorboard

### PR DESCRIPTION
**Pull request by quic-swatia**
Plotting the correct (scaled) loss on tensorboard from the training.
Removed this change for now: Plotting the loss for each sample of eval on tensorboard rather the average.

### CODE REVIEW

File: QEfficient/finetune/utils/train_utils.py [hunk 1]
```diff
@@ -222,12 +221,14 @@ def train(
                 dist.barrier()
                 dist.all_reduce(eval_epoch_loss, op=dist.ReduceOp.SUM)
                 if local_rank == 0:
-                    tensorboard_updates.add_scalars("loss", {"val": eval_epoch_loss}, total_train_steps)
+                    for i in range(len(temp_val_loss)):
+                        tensorboard_updates.add_scalars("loss", {"eval": temp_val_loss[i]}, i)
```
Comment (quic-mamta) [inline QEfficient/finetune/utils/train_utils.py:225]: This will not work as it will plot the values of eval step in same range (0,len(temp_val_loss)) for each epoch.

Comment (quic-mamta) [inline QEfficient/finetune/utils/train_utils.py:225]: This will not work as it will plot the values of eval step in same range (0,len(temp_val_loss)) for each epoch.

**Review by quic-mamta — CHANGES_REQUESTED** [review]

**Review by quic-mamta — APPROVED** [review]

**Comment by quic-amitraj** [comment]
Please change the timeout for `Non CLI and non QAIC` stage in Jenkins from 10 to 15.

**Review by vbaddi — APPROVED** [review]
